docs/react-testing-library/example-intro.md

```diff
@@ -117,13 +117,15 @@ fetch.js
 
 ```jsx
 import React,{useState} from 'react'
+import axios from 'axios'
 
 export default function Fetch({url}) {
-  const {greeting, setGreeting} = useState('')
-  const {buttonClicked, setButtonClicked} = useState(false)
+  const [greeting, setGreeting] = useState('')
+  const [buttonClicked, setButtonClicked] = useState(false)
   
   const fetchGreeting = async () => {
-    const data = await fetch(url)
+    const response = await axios.get(url)
+    const data = await response.data
```

Comment:
I don't think we need `await` here.

Reply:
Oh yeah! Forget to remove as it had "fetch" before so were using for response.json().